pkgs/applications/misc/gcal/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, pkgconfig, ncurses, perl, less }:
+
+stdenv.mkDerivation rec {
+  name = "gcal-${version}";
+  version = "4.1";
+
+  src = fetchurl {
+    url = "https://ftp.gnu.org/gnu/gcal/${name}.tar.xz";
+    sha256 = "1av11zkfirbixn05hyq4xvilin0ncddfjqzc4zd9pviyp506rdci";
+  };
+
+  enableParallelBuilding = true;
+
+  buildInputs = [ ncurses perl less ];
```

Comment:
Does the build need `perl` and `less`?

Reply:
> Does the build need perl and less?

Not necessarily. It works without them.

I do not know exactly why `perl` is checked at configuration time.

`less` is used as a default pager. But there are other ways of using a pager, described in the [manual](https://www.gnu.org/software/gcal/manual/gcal.html). Just look at the `--pager` option. For instance setting the environment variable `PAGER`.